dotcom-rendering/playwright/tests/parallel-1/article.embeds.e2e.spec.ts

```diff
@@ -0,0 +1,171 @@
+import type { Page } from '@playwright/test';
+import { expect, test } from '@playwright/test';
+import { waitForIsland } from 'playwright/lib/islands';
+import { expectToBeVisible } from 'playwright/lib/locators';
+import { cmpAcceptAll } from '../../lib/cmp';
+import { loadPage } from '../../lib/load-page';
+
+const getIframeBody = async (page: Page, iframeSelector: string) => {
+	const iframeBodyLocator = page.frameLocator(iframeSelector).locator('body');
+	await expect(iframeBodyLocator).toBeAttached({ timeout: 10000 });
+	return iframeBodyLocator;
+};
+
+test.describe('Embeds', () => {
+	test.describe('AMP', () => {
+		test('should render the corona interactive atom embed', async ({
+			page,
+		}) => {
+			await loadPage(
+				page,
+				'/AMPArticle/https://www.theguardian.com/world/2020/apr/24/new-mother-dies-of-coronavirus-six-days-after-giving-birth',
+			);
+			await cmpAcceptAll(
+				page,
+				'amp-consent > iframe[src*="sourcepoint"]',
+			);
+
+			await expect(
+				await getIframeBody(
+					page,
+					'amp-iframe[data-testid="atom-embed-url"] > iframe',
+				),
+			).toContainText('Daily cases');
+		});
+
+		test('should render the counted interactive embed', async ({
+			page,
+		}) => {
+			await loadPage(
+				page,
+				'/AMPArticle/https://www.theguardian.com/us-news/2015/nov/05/police-tasers-deaths-the-counted',
+			);
+			await cmpAcceptAll(
+				page,
+				'amp-consent > iframe[src*="sourcepoint"]',
+			);
+
+			const ampIframeContainerSelector =
+				'amp-iframe[src="https://interactive.guim.co.uk/embed/2015/10/2015-10-counted-table/"]';
+
+			await page
+				.locator(ampIframeContainerSelector)
+				.scrollIntoViewIfNeeded();
+
+			const ampIframeSelector = `${ampIframeContainerSelector} > iframe`;
+
+			await expect(
+				await getIframeBody(page, ampIframeSelector),
+			).toContainText('Deaths after Taser use: the findings');
+		});
+	});
+
+	test.describe('WEB', function () {
```

Comment:
not necessarily for this migration, but what do you think about adding `apps` tests? might not be necessary but just a thought!

Reply:
Good idea! I'll try this out as a separate PR